components/mail/sendmail/Makefile

```diff
@@ -21,27 +21,27 @@
 
 #
 # Copyright (c) 2012, 2016, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2022 Friedrich Kink
 #
 
-BUILD_BITS=     64
-USE_OPENSSL11=	yes
+USE_OPENSSL11=		yes
 
+BUILD_STYLE=		justmake
+BUILD_BITS=		64_and_32
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME=		sendmail
-COMPONENT_VERSION=	8.17.1
-COMPONENT_REVISION=	0
-COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_PROJECT_URL=	ftp://ftp.sendmail.org/pub/sendmail/
-COMPONENT_ARCHIVE=	$(COMPONENT_NAME).$(COMPONENT_VERSION).tar.gz
-COMPONENT_ARCHIVE_HASH=	sha256:04bc76b6c886e6d111be7fd8daa32b8ce00128a288b6b52e067bc29f3854a6e6
-COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)$(COMPONENT_ARCHIVE)
+COMPONENT_NAME=         sendmail
+COMPONENT_VERSION=      8.17.1
+COMPONENT_REVISION=     1
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL=  ftp://ftp.sendmail.org/pub/sendmail/
+COMPONENT_ARCHIVE=      $(COMPONENT_NAME).$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:04bc76b6c886e6d111be7fd8daa32b8ce00128a288b6b52e067bc29f3854a6e6
+COMPONENT_ARCHIVE_URL=  $(COMPONENT_PROJECT_URL)$(COMPONENT_ARCHIVE)
 
 TPNO=			23958
 
```

Comment:
Please remove the TPNO (and also from manifest file if referenced), it's an Oracle thing.